src/main/java/software/amazon/awssdk/crt/io/TlsContextOptions.java

```diff
@@ -136,6 +139,17 @@ public void initMTLSFromPath(String certificatePath, String privateKeyPath) {
         tlsContextOptionsInitMTLSFromPath(getNativeHandle(), certificatePath, privateKeyPath);
     }
 
+    /**
+     * Sets the path to the certificate that identifies this TLS host. Must be in
+     * PEM format.
+     * 
+     * @param certificate PEM format certificate
+     * @param privateKey  PEM format private key
+     */
+    public void initMTLS(ByteBuffer certificate, ByteBuffer privateKey) {
```

Comment:
I like the "FromX" Suffix on these readability wise.  Maybe FromBuffer(s)?

Reply:
On second thought it seems like this mirrors the aws-c-io functions.  Maybe not worth changing.